recipes/onnx2torch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f668f6ecb2f34029d4c85a66be2f4a329a3d5d2ed8513395e508322d20dcb859
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

Comment:
This can probably be noach if the failure on Windows is just a missing dependency.

Reply:
:heavy_check_mark:

Reply:
@ocefpaf I added `noarch: python` but build is failed on win. What have I done wrong?

Reply:
Like I mentioned above, if the failure is a missing dependency, they it is fine. What failed?

Reply:
There is no `torchvision`:
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("torchvision[version='>=0.9.0']")}
```

Reply:
Yep. So it is fine to leave it as noarch.

Reply:
Okay, thanks for your help.